src/AltinnCore/Runtime/Controllers/InstanceController.cs

```diff
@@ -195,7 +195,7 @@ public async Task<IActionResult> CompleteAndSendIn(string org, string service, G
                     InstanceId = instance.Id,
                     InstanceOwnerId = instance.InstanceOwnerId.ToString(),
                     UserId = requestContext.UserContext.UserId,
-                    WorkflowStep = instance.CurrentWorkflowStep
+                    WorkflowStep = instance.Workflow.CurrentStep,
```

Comment:
remove ,

Reply:
Stylecop give warning on missing , but syntactic you are right.